en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+[silverstripe/recipe-collaboration](https://github.com/silverstripe/recipe-collaboration) | 2
+[silverstripe/recipe-content-blocks](https://github.com/silverstripe/recipe-content-blocks) | 3
+[silverstripe/recipe-form-building](https://github.com/silverstripe/recipe-form-building) | 2
+[silverstripe/recipe-reporting-tools](https://github.com/silverstripe/recipe-reporting-tools) | 2
+[silverstripe/recipe-services](https://github.com/silverstripe/recipe-services) | 2
+[silverstripe/registry](https://packagist.org/packages/silverstripe/registry) | 3
+[silverstripe/restfulserver](https://packagist.org/packages/silverstripe/restfulserver) | 3
+[silverstripe/securityreport](https://packagist.org/packages/silverstripe/securityreport) | 3
+[silverstripe/segment-field](https://packagist.org/packages/silverstripe/segment-field) | 3
+[silverstripe/sharedraftcontent](https://packagist.org/packages/silverstripe/sharedraftcontent) | 3
+[silverstripe/sitewidecontent-report](https://packagist.org/packages/silverstripe/sitewidecontent-report) | 4
+[silverstripe/spamprotection](https://packagist.org/packages/silverstripe/spamprotection) | 4
+[silverstripe/staticpublishqueue](https://packagist.org/packages/silverstripe/staticpublishqueue) | 6
+[silverstripe/subsites](https://packagist.org/packages/silverstripe/subsites) | 3
+[silverstripe/tagfield](https://packagist.org/packages/silverstripe/tagfield) | 3
+[silverstripe/taxonomy](https://packagist.org/packages/silverstripe/taxonomy) | 3
+[silverstripe/textextraction](https://packagist.org/packages/silverstripe/textextraction) | 4
+[silverstripe/totp-authenticator](https://packagist.org/packages/silverstripe/totp-authenticator) | 5
+[silverstripe/userforms](https://packagist.org/packages/silverstripe/userforms) | 6
+[silverstripe/versionfeed](https://packagist.org/packages/silverstripe/versionfeed) | 3
+[silverstripe/webauthn-authenticator](https://packagist.org/packages/silverstripe/webauthn-authenticator) | 5
+[symbiote/silverstripe-advancedworkflow](https://packagist.org/packages/symbiote/silverstripe-advancedworkflow) | 6
+[symbiote/silverstripe-gridfieldextensions](https://packagist.org/packages/symbiote/silverstripe-gridfieldextensions) | 4
+[symbiote/silverstripe-multivaluefield](https://packagist.org/packages/symbiote/silverstripe-multivaluefield) | 6
+[symbiote/silverstripe-queuedjobs](https://packagist.org/packages/symbiote/silverstripe-queuedjobs) | 5
+[tractorcow/silverstripe-fluent](https://packagist.org/packages/tractorcow/silverstripe-fluent) | 7
+
+## Supported NPM packages
+
+The following two NPM packages are also supported because they are required to build the UI of Silverstripe CMS 5:
+- [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config)
+- [@silverstripe/eslint-config](https://www.npmjs.com/package/@silverstripe/eslint-config)
+
+You'll only need to interact with them if you are customising the Silverstripe CMS UI.
```

Comment:
This sentence seems to suggest that customising the UI is uncommon, which it isn't.
I'd honestly recommend just leaving this sentence out altogether, but if you want to keep it, it should probably sound less opinionated and link to the relevant docs.

Reply:
Be gone sentence!